components/menu/src/auro-menu.js

```diff
@@ -185,7 +185,7 @@ export class AuroMenu extends LitElement {
       this.items.forEach((item) => {
         item.classList.remove('active');
         item.removeAttribute('selected');
-        item.removeAttribute('aria-selected');
+        item.setAttribute('aria-selected', 'false');
```

Comment:
with the line above, line 188 should not be needed as menuoption's updated function should take care of this.

Reply:
Removed